src/generic.rs

```diff
@@ -102,7 +102,8 @@ impl<R: CryptoRngCore, S: SieveFactory> Iterator for SieveIterator<'_, R, S> {
 
 #[cfg(test)]
 mod tests {
-    use rand_core::{CryptoRngCore, OsRng};
```

Comment:
What happened to `OsRng` in the new release?

Reply:
`OsRng` not longer implements `CryptoRng` but only `TryCryptoRng` but I completely missed this [`UnwrapErr`](https://docs.rs/rand_core/latest/rand_core/struct.UnwrapErr.html) which would bring it back.

I migrated away from `OsRng` but I guess I can move back to `UnwrapErr(OsRng)` at least for `default_rng`.
I think `rand_chacha` is fine for tests though. What's your opinion?

Reply:
meh, `UnwrapErr` does not care about CryptoRng though :(

Reply:
`default_rng` would certainly be better with `OsRng`, but the feature is actually going away soon (#62).

As for the tests, it's most probably fine, I just don't want to introduce new entities, and writing `UnwrapErr(OsRng)` is no worse than `ChaChaRng::from_os_rng()`.

Reply:
I wish they'd make an alias for that in `rand_core`, actually.